tools/demo.py

```diff
@@ -133,11 +137,13 @@ def parse_args():
         net = resnetv1(num_layers=101)
     else:
         raise NotImplementedError
-    net.create_architecture(21,
-                          tag='default', anchor_scales=[8, 16, 32])
-
+    print('create_architecture; ')
+    net.create_architecture(int(21),
+                          tag='default', anchor_scales=[int(8), int(16), int(32)])
+    print('load_state_dict: ')
```

Comment:
Why do you need convert the numbers to int? Is this caused by the python version?